global/hooks/useEnvironmentalData/index.ts

```diff
@@ -139,7 +140,14 @@ const useEnvironmentalData = (origin: string) => {
 				signal,
 			});
 
-			if (!submissionResponse?.data || !('inserts' in submissionResponse.data)) {
+			if (
+				!submissionResponse?.data ||
+				!(
+					'inserts' in submissionResponse.data ||
+					'updates' in submissionResponse.data ||
+					'deletes' in submissionResponse.data
+				)
```

Comment:
this could also be expressed as (or similar, if `status` is not the right varname).
```suggestion
				!['inserts', 'updates', 'deletes'].some((status) =>  status in submissionResponse.data)
```

the strings could also be abstracted into reused constants (e.g. `SUBMISSION_STATUSES.INSERT`), imported everywhere from a single place, so that if at any point the implementation changes, we don't have go hunting for e.g. 'inserts' everywhere).

Reply:
Updated the code to iterate over the constant values of `EventTypeToKey` and use the `.some` method as mentioned. Really appreciate the suggestions!